### PR TITLE
Add citation info to faq

### PR DESCRIPTION
**Pull request by beets**
image

### CODE REVIEW

**Review by shifucun — APPROVED** [review]

File: server/templates/static/faq.html [hunk 1]
```diff
@@ -180,6 +180,26 @@ <h1>Frequently Asked Questions </h1>
...
+    <blockquote>
+    Data Commons {{ current_year }}, <i>Data Commons</i>, viewed {{ current_date }}, &#60;https://datacommons.org/&#62;.
+    </blockquote>
+
+  <p>
+    If citing data from a particular dataset, e.g. CDC Places, then use:
+  </p>
+
+    <blockquote>
+    Data Commons {{ current_year }}, CDC Places, electronic dataset, <i>Data Commons</i>, viewed {{ current_date }}, &#60;https://datacommons.org/&#62;.
```
Comment (shifucun) [inline server/templates/static/faq.html:197]: remove the trailing slash? It will redirect to get rid of it anyway

Comment (beets) [reply server/templates/static/faq.html]: done!

**Comment by beets** [comment]
thanks for the review!

**Comment by shifucun** [comment]
Should we merge this PR?

**Comment by beets** [comment]
thanks for the reminder - i missed that the lint check failed..

**Pull request merged**